external/netcoreapp/netcoreapp.depproj

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please add a comment to this project describing its purpose: to download a targeting pack only for a specific NETCoreApp version.